src/kudu/util/net/sockaddr.cc

```diff
@@ -103,11 +105,11 @@ std::string Sockaddr::ToString() const {
 }
 
 bool Sockaddr::IsWildcard() const {
-  return addr_.sin_addr.s_addr == 0;
+  return addr_.sin6_addr.s6_addr == 0;
 }
 
 bool Sockaddr::IsAnyLocalAddress() const {
-  return (NetworkByteOrder::FromHost32(addr_.sin_addr.s_addr) >> 24) == 127;
+  return (NetworkByteOrder::Load128(addr_.sin6_addr.s6_addr) == 1);
```

Comment:
Did you borrow this implementation from somewhere?
Are we sure this is correct.

Reply:
I guess its trying to check whether it is ::1 , so makes sense.